views/dashboard.handlebars

```diff
@@ -1,18 +1,22 @@
-<div>
-    <h3>Post</h3>
-
-<div>
-
-    <p>some text</p>
-
-    <ul>
-        {{#each user.post as |post|}}
-        <li>
-            <h4> {{post.book_title}}</h4>
-            <p> {{post.book_author}}</p>
-            <p> {{post.rating}}</p>
-        </li>
-{{/each}}
-    </ul>
+<div class="dashContainer">
+    <div class="textContainer">
+        <!-- This will be for the logged in user's personal dashboard -->
+        <p>Welcome,</p>
+        <h1> {{book.user.username}} </h1>
```

Comment:
I think for this h1 we can leave it as user.username, since we are rendering the user model to this view, we are just including the book, and post models.